tests/system.rs

```diff
@@ -1285,29 +1285,37 @@ fn keeps_newlines_in_external_mod_doc() {
     assert_eq!("The mods multiline documentation\n\nwith an empty line", got.docs);
 }
 
+/// Addresses https://github.com/racer-rust/racer/issues/618
 #[test]
-fn issue_618() {
+fn always_get_all_doc_lines() {
```

Comment:
It's kind of nice to have the issue number in the test name. I like what you've done to explain what the test actually is though. Maybe we can combine them? `issue_618_always_foo`.